articles/connectivity/conn-faq.md

```diff
@@ -141,19 +139,19 @@ Yes, UKCloud does support an RLI connection for customers who have been already
 
 ## If I have an MPLS circuit direct into UKCloud, do I still have to go through a PSN connection to access the Elevated Portal?
 
-No, you don't need a PSN connection as you can access the Portal from within a VM on the Elevated platform.
+No, you do not need a PSN connection as you can access the Portal from within a VM on the Elevated platform.
 
 ## Are external Domain Name System (DNS) services available?
 
-No, we don't currently offer this service. Customers can implement their own DNS servers within their solution, or configure their virtual firewall to enable connectivity to an externally hosted DNS server (for example, those hosted on Government Secure Networks such as PSN, or those available on the internet such as Google 8.8.8.8).
+No, we do not currently offer this service. Customers can implement their own DNS servers within their solution, or configure their virtual firewall to enable connectivity to an externally hosted DNS server (for example, those hosted on Government Secure Networks such as PSN, or those available on the internet such as Google 8.8.8.8).
 
 ## Does UKCloud support Janet connectivity?
 
-Yes, UKCloud provides a Janet connection provided free for our customers. In order to utilise this network, you will need to be authorised by Jisc. There is no inbound or outbound usage charge for using the Janet connection to access the UKCloud platform.
+Yes, UKCloud provides Janet connectivity free for our customers. In order to utilise this network, you will need to be authorised by Jisc. There is no inbound or outbound usage charge for using the Janet connection to access the UKCloud platform.
```

Comment:
Thought we were moving from Jisc?

Reply:
Leaving in for now until I can confirm with SME. We do still provide instructions that customer needs to get authorisation from Jisc.